src/app/backend/handler/apihandler.go

```diff
@@ -816,13 +850,14 @@ func (apiHandler *APIHandler) handleDeploy(request *restful.Request, response *r
 }
 
 // Handles deploy from file API call.
-func (apiHandler *APIHandler) handleDeployFromFile(request *restful.Request, response *restful.Response) {
+func (apiHandler *APIHandler) handleDeployFromFile(apiClient *clientK8s.Clientset, request *restful.Request, response *restful.Response) {
 	deploymentSpec := new(deployment.AppDeploymentFromFileSpec)
 	if err := request.ReadEntity(deploymentSpec); err != nil {
 		handleInternalError(response, err)
 		return
 	}
 
+	// TODO inject bearer token
```

Comment:
Hmm... Can you explain the problem? Cannot you pass the apiClient as you did in other places in the code?